mkt/developers/forms.py

```diff
@@ -701,8 +714,8 @@ def save(self, addon, commit=True):
 
             remove_icons(destination)
             tasks.resize_icon.delay(upload_path, destination,
-                                    mkt.CONTENT_ICON_SIZES,
-                                    set_modified_on=[addon])
+                                    mkt.CONTENT_ICON_SIZES)
+            addon.update(modified=datetime.now())
```

Comment:
How is the `icon_hash` going to be set ? It calls `pngcrush_image` task, but since `set_modified_on` no longer exists, the `icon_hash` is never going to be set I'm afraid. (and there might be other calls to `resize_icon` with the same issue)